apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.html.ts

```diff
@@ -114,19 +116,39 @@ export function AdministratorFeature({
                 Grant feature
               </button>
             </div>
-            ${featureGrants.length > 0
+            ${featureInConfig != null
               ? html`
                   <div class="list-group list-group-flush">
+                    <div class="list-group-item">
```

Comment:
Not that it _really_ matters for this internal page, but two adjacent `list-group` elements produce a double border between them; if this were public-facing, I'd ask you to render both the explicit grants and the global config setting in the same `list-group`.

<img width="1127" alt="Screenshot 2023-10-03 at 13 30 58" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/5a3cb5a2-bbe9-4028-8552-5abed794639d">

Reply:
Done in 20d91108e.